src/battle/action_cmd/power_shock.c

```diff
@@ -148,8 +147,10 @@ void func_802A9310_42D220(void) {
                     actionCommandStatus->hudElementX,
                     actionCommandStatus->hudElementY + 28);
 
-                temp_v0_2 = (((100 - actionCommandStatus->unk_46) * 60) / 100);
-                temp_v0_2 = 29 - temp_v0_2;
+                //temp_v0_2 = (((100 - actionCommandStatus->unk_46) * 60) / 100);
```

Comment:
```suggestion
```

Reply:
It's in NON_MATCHING so this is just there to indicate what the line should really look like (though it's less nice of a score)...actually yeah I'll just remove it